config/schedule.rb

```diff
@@ -0,0 +1,6 @@
+# Use this file to easily define all of your cron jobs.
+#
+set :output, "/tmp/cron_log.log"
+every 1.day, at: '4:00 am' do
+  rake "db:sessions:trim"
```

Comment:
This rake task and the cron job don't output anything if it runs successfully. So for MMT Hoan created a different rake task that invokes this one and also adds something to the logs so we know it was run successfully. We probably want to do the same thing here.